app/partials/index.html

```diff
@@ -1,13 +1,100 @@
 <div class="container">
-  <div class="card-deck">
-    <div ng-repeat="rdesc in apidesc">
-      <div class="card" style="width: 230px; height: 150px; margin: 20px">
-        <div class="card-body">
-          <h4 class="card-title"><b>{{rdesc.attributes.name.plural}}</b></h4>
-          <a ui-sref="list({resourceName: rdesc.id})" class="btn btn-outline-primary">List</a>
-          <a ui-sref="add({resourceName: rdesc.id})" class="btn btn-success">Add</a>
-        </div>
-      </div>
+    <div class="card-deck">
+        <div class="card top-card">
+            <h4 class="card-header">
+                Sites
+            </h4>
+            <div class="card-body">
+                <ul class="list-group list-group-flush">
+                    <div ng-repeat="site in relatedSites">
+                        <li class="list-group-item">
+                            {{site.name}}
+                            <a ng-href="{{site.prodLink}}">(prod)</a>
+                            <a ng-href="{{site.stagingLink}}">(staging)</a>
```

Comment:
some sites may not have staging, so rmb to hcekcing if ` {{site.stagingLink}}` exists first before rendering the staging link. also i wonder if `(test)` or `(beta)` would be a better / clearer phrase to display instead of `(staging)`?